server/routes/orders.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const mysql = require("mysql");
+
+const pool = mysql.createPool({
+    connectionLimit: 10,
+    host: "localhost",
+    user: "geoffrey",
+    password: "Ya9pQcM3x7BsRqDX",
+    database: "toyzrus"
+});
+
+var router = express.Router();
+
+//serve all orders
+router.get("/", (req, res) => {
+    const allItemsString = 'SELECT * FROM orders;';
+    pool.getConnection((err, connection) => {
+        if (err) {console.log(err.message); throw err;}
+        connection.query(allItemsString, function(error, results) {
+            connection.release;
+            if (error) {console.log(error.message); throw error;}
+            else {
+            console.log(results);
+            res.json(results);
+            }
+        });
+    });
+});
+
+//change status of orderId record to "cancelled"
+router.get("/cancel:orderId", (req, res) => {
```

Comment:
This would match on url's like `/cancel56`, but for this kind of route we would expect the id to come in the body of the req. We also need to make sure that any route updating the data is a post and not a get. Gets should only be used when retrieving data.